plugin.cpp

```diff
@@ -188,7 +188,33 @@ static const char *default_config = QUOTE({
             "displayName" : "Debug Trace File",
 	    "group" : "OPC UA Advanced",
             "order" : "19"
-            }
+            },
+	"filterRegex" : {
+		"description" : "Regular expression for filtering nodes that are to be processed",
+		"type" : "string",
+		"default" : "",
+		"displayName" : "Regular expression",
+		"order" : "20",
+		"group" : "Filtering"
+		},
+	"filterScope" : {
+		"description" : "Specify the scope of the filter",
+		"type": "enumeration",
+		"options":["Object", "Variable", "Object and Variable"],
+		"default" : "Variable",
+		"displayName" : "Scope",
+		"order" : "21",
+		"group" : "Filtering"
+		},
+	"filterAction" : {
+		"description" : "Filtering action to perform",
+		"type": "enumeration",
+		"options":["Include nodes", "Exclude nodes"],
```

Comment:
I'm not sure the use of nodes is right here. Given that the scope could be "Variable", and is by default, what are we including or excluding, a node or a variable?

Probably just as easy to just use "Include" and "exclude", the scope tells you what you are including.